package.json

```diff
@@ -111,12 +111,14 @@
     "@rjsf/bootstrap-4": "^2.3.0",
     "@rjsf/core": "^2.3.0",
     "ajv": "^6.12.3",
+    "apexcharts": "^3.22.1",
     "axios": "^0.19.2",
     "chart.js": "^2.9.3",
     "closest": "0.0.1",
     "core-js": "^3.6.5",
     "dagre": "^0.8.5",
     "dayjs": "^1.8.34",
+    "google-palette": "^1.1.0",
```

Comment:
What do you mean by 'in line with the rest of the dashboard'? This is used to generate a list of colors where each color is adequately different from the previous one.

Reply:
everywhere in the dashboard, when we refer to disconnected devices, we use a specific color (a specific shade of red) same goes for connected devices. To be consistent with the look and feel, we must respect those choices.
Same goes for overall lightness and saturation. Shifting too much from what it's being used, and the object will popup a lot in the context.

Reply:
I see what you mean now. Indeed, I did already copy over the same colors for Connected/Disconnected devices. I wasn't aware of the choice of hsl values so I used google-palette since it generates a proper random palette of colors, not a varying scale of colors. However now that I saw how you do it in PR #230 I will use the same logic to generate colors for the generic Pie Chart